src/ast/ast_build_op_contexts.c

```diff
@@ -157,8 +158,8 @@ void AST_PreparePathCreation(const cypher_astnode_t *path, QueryGraph *qg, rax *
 	}
 }
 
-AST_MergeContext AST_PrepareMergeOp(const cypher_astnode_t *merge_clause, QueryGraph *qg,
-									rax *bound_vars) {
+AST_MergeContext AST_PrepareMergeOp(const cypher_astnode_t *merge_clause, GraphContext *gc,
```

Comment:
Can't we get `QueryGraph` from `GraphContext` ?

Reply:
No, the `QueryGraph` lives on the `ExecutionPlan`.